Samples/Windowing/cs-winforms-unpackaged/winforms_unpackaged_app.sln

```diff
@@ -0,0 +1,37 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.31229.75
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "winforms_unpackaged_app", "winforms_unpackaged_app.csproj", "{D42797AB-4CBE-44A3-A9F6-5EAF2D657C3D}"
+EndProject
+Global
+	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|arm64 = Debug|arm64
```

Comment:
we should remove all arm64 configs here as well, since winforms doesn't support it.